### PR TITLE
feat: create calendar card

### PR DESCRIPTION
**Pull request by nguyend-nam**

### CODE REVIEW

File: components/CalendarCard/CalendarCard.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import { IconLocationDotSolid } from 'components/icons/components/IconLocationDotSolid'
+import { IconEllipsisHSolid } from 'components/icons/components/IconEllipsisHSolid'
+
+interface UserProps {
+  name: string
+  avatar: string
+  date: string
+  time: string
+  location: string
+  isFirstLoading?: boolean
```
Comment (zlatanpham) [inline components/CalendarCard/CalendarCard.tsx]: The skeleton UI should be in a separate component. However, it is out of the scope of the task so I would say we should get rid of the part.

File: components/CalendarCard/CalendarCard.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import { storiesOf } from '@storybook/react'
+import { CalendarCard } from './CalendarCard'
+
+const data = [
```
Comment (zlatanpham) [inline components/CalendarCard/CalendarCard.stories.tsx:4]: Did you use all of the records? Should only declare what is actually being used.

File: components/CalendarCard/CalendarCard.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        </div>
+      ) : (
+        <div className="flex items-center p-1 pt-5 pb-5">
+          <img
+            className="rounded-full h-14 w-14 mr-2.5"
+            src={avatar}
+            alt={name}
+          />
+
+          <div className="mr-3 ml-3 w-full text-sm">
```
Comment (zlatanpham) [inline components/CalendarCard/CalendarCard.tsx]: Is `w-full` needed? since the default width of a div is always 100%

**Review by zlatanpham — CHANGES_REQUESTED** [review]

File: components/CalendarCard/CalendarCard.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <div>
+      {data.map((user) => (
+        <CalendarCard
+          name={user.name}
+          avatar={user.avatar}
+          date={user.date}
+          time={user.time}
+          location={user.location}
+        />
```
Comment (zlatanpham) [inline components/CalendarCard/CalendarCard.stories.tsx:25]: Mising `key` prop

**Review by zlatanpham — APPROVED** [review]
One nit

File: components/CalendarCard/CalendarCard.stories.tsx (hunk 4 above)
Comment (nguyend-nam) [reply components/CalendarCard/CalendarCard.stories.tsx:25]: Ok I'll fix it right now in the latest Pr

**Pull request merged**